tests/data/jobDef.ts

```diff
@@ -51,6 +51,47 @@ export const getBuildJobPlain = (): Job => ({
   } as Payload,
 });
 
+export const getStagingJobDef = (): Job => ({
```

Comment:
For some reason, we were using a `Job` representative of a deployment in `stagingJobHandler` tests. Created this separate function to be more accurate and to help pass the new test